### ISSUE TITLE
Extraction of DATA.WAR fails for unknown reason

### ISSUE THREAD

**Issue by pVVEN** [Wargus/war1gus#367]
I tried running the extractor on my DOS copy of DATA.WAR, which I put in the same folder, but it failed with an error:

> Extraction failed!
> The following command was used to extract the data (you can run it manually in a console to find out more): @E:\Games\War1gus\war1tool.exe -v -r E:\Games\War1gus E:\Games\War1gus

When I tried running that command in command prompt (with admin privileges), it just gave me this:

> war1tool V3.0.0.20201210 for Stratagus, (c) 1998-2015 by The Stratagus Project.
>         Written by Lutz Sammer, Nehal Mistry, Jimmy Salmon, Pali Rohar, and Tim Felgentreff.
>         https://github.com/Wargus/war1gus
> 
> Usage: E:\Games\War1gus\war1tool.exe [-m] [-v] [-V] [-h] archive-directory [destination-directory]
>         -m      Extract and convert midi sound files (may take some time)
>         -v      Extract and convert videos
>         -V      Show version
>         -h      Show usage (this text)
> archive-directory       Directory which includes the archives (data.war...)
> destination-directory   Directory where the extracted files are placed (default: data.wc1).
> 

I tried the command again without the -r flag, and as far as I can tell it was successful:

> E:\Games\War1gus>@E:\Games\War1gus\war1tool.exe -v E:\Games\War1gus E:\Games\War1gus
> Extract from "E:\Games\War1gus/" to "E:\Games\War1gus"
> Please be patient, the data may take a couple of minutes to extract...
> 
> E:\Games\War1gus>

But it still asks to extract data when I try running the game. If it helps, here's the MD5 of my DATA.WAR file:

E91669CA9C8D007C88067518B9A5452E

 - OS: Windows 10
 - Version 3.2.0 portable

**Comment by pVVEN** [comment]
Per the dev's suggestion on my Wargus thread for a similar issue, I tried using a mounted image of the CD instead. It seemed to extract everything successfully. Very strange, as the DATA.WAR file I used before was pulled from that image. I compared the MD5s again to be sure. Maybe the extractor tool is too specific in what it's looking for? Well the issue is seemingly resolved for me on this one so I'll close the ticket and make a backup of the extracted files.
